share/messages_to_error_log.txt

```diff
@@ -3914,6 +3914,9 @@ ER_CONN_CONTROL_FAILED_TO_SET_CONN_DELAY
 ER_CONN_CONTROL_FAILED_TO_UPDATE_CONN_DELAY_HASH
   eng "Failed to update connection delay hash for account : %s"
 
+ER_CONN_CONTROL_FAILED_CONN_THRESHOLD_REACHED
+  eng "Failing connection attempts threshold reached for %s, %i delayed connection attempts during last %i seconds"
```

Comment:
`%i` is an incorrect formatter for uint64_t
it should be `PRIu64`
```
"Failing connection attempts threshold reached for %s, %" PRIu64 " delayed connection attempts during last %" PRIu64 " seconds"
```
not sure though if such lines can be inserted into `messages_to_error_log.txt`

Reply:
%llu will be used in message template

Reply:
We should never introduce error messages in middle of ER_. it should be always last. Otherwise, other codes will see a shift in number.

Reply:
Moved messages to the end